controlplane/kubeadm/internal/controllers/controller.go

```diff
@@ -587,15 +591,27 @@ func (r *KubeadmControlPlaneReconciler) reconcileClusterCertificates(ctx context
 	return nil
 }
 
+// scope holds the different objects that are read and used during the reconcile.
+type scope struct {
```

Comment:
What I'm doing in other PRs is to use controlPlane as a scope (and if fact this is what it is).
Let's keep doing this without introducing a new scope.

Reply:
Okay, fine for me. Adding deletingReason & deletingMessage just seemed pretty strange to me. Considering what we store within ControlPlane today

Reply:
Done